website/docs/r/domain.html.markdown

```diff
@@ -551,6 +552,17 @@ boot_device {
 }
 ```
 
+### Add tags to the domain
+
+By setting tags for the domain you can more easily filter and export the terraform state to external tools:
+
+```hcl
+tags {
```

Comment:
It would improve the Ux if we add an example to our example dir with a domain use case, create a dir https://github.com/dmacvicar/terraform-provider-libvirt/tree/master/examples

Reply:
I think it would make more sense to just expand upon the coreos example with extra tags. Its not such a wide reaching feature that it would require an extra example in my opinion.

i.e. expand the example to add 1-2 master tags and the rest worker or something to show how it can be used.

Reply:
Amended it to the commit with the CoreOS example

Reply:
thx!